src/diffusers/loaders/lora_base.py

```diff
@@ -349,6 +367,8 @@ def _load_lora_into_text_encoder(
     # Load the layers corresponding to text encoder and make necessary adjustments.
     if prefix is not None:
         state_dict = {k.removeprefix(f"{prefix}."): v for k, v in state_dict.items() if k.startswith(f"{prefix}.")}
+        if metadata is not None:
```

Comment:
For future: `_load_lora_into_text_encoder()` and `load_lora_adapter()` share a bunch of common logic which I will refactor in a future PR.